src/Sylius/Bundle/CoreBundle/Behat/MetadataContext.php

```diff
@@ -342,7 +340,7 @@ private function getFormWithFields(ElementInterface $element, array $fields)
             }
         }
 
-        return null;
+        return;
```

Comment:
@bendavies Yes, let's remove this rule for now and rebase pls, I will merge asap, thanks!